libs/designsystem/src/lib/components/segmented-control/segmented-control.component.scss

```
@@ -21,32 +21,62 @@
     }
   }
 
+  &.default-mode {
+    ion-segment {
+      --background: #{utils.get-color('white')};
+
+      display: inline-flex; // The segmented control itself is block-level, but we don't want the background to stretch
+      width: auto; // The segmented control itself is block-level, but we don't want the background to stretch
+      overflow: visible; // Ensures the badge is not cut off
+      contain: unset; // Ensures the badge is not cut off
+    }
+  }
+
   &.chip-mode {
-    display: flex;
-    flex-wrap: nowrap;
-    overflow-x: auto;
-    overflow-y: hidden; // chip touch area height is bigger than segmented control height
-    // Hide scrollbar on touch device:
-    @include utils.touch {
-      scrollbar-width: none; /* Firefox */
-      &::-webkit-scrollbar {
-        display: none;
+    ion-segment {
+      @include utils.touch {
+        scrollbar-width: none; /* Firefox */
+        &::-webkit-scrollbar {
+          display: none;
+        }
+      }
+
+      --background: none;
+
+      border-radius: 0;
+      padding: #{utils.size('xxxs')} #{utils.size('xxxs')};
```

If you are declaring the same value for both axises anyway then:

```suggestion
      padding: #{utils.size('xxxs')};
```

---

Great, didn't catch that when fiddling with the padding!